packages/hooks/use-disclosure/src/index.ts

```diff
@@ -92,12 +107,16 @@ export const usePromiseDisclosure = <
   error,
   ...rest
 }: UsePromiseDisclosureProps<Y, M> = {}) => {
-  const {
+  let {
     isOpen,
+    open,
     onClose: onInternalClose,
     onOpen: onInternalOpen,
     onToggle,
   } = useDisclosure<Y, M>(rest)
+
+  open = isOpen
+
```

Comment:
@kaehehehe 
Sorry! Just checking around. Since the PR has merged, this might be random, but may I ask if this is change was intentional? Shouldn we use `??=` operator instead?

Reply:
@nathanlao 
The `isOpen` can be `undefined`, so I can't use the `??=` operator. Therefore, I always assign the value of `isOpen` to the `open` variable.